pyproject.toml

```diff
@@ -75,6 +75,9 @@ select = [
     "D",  # see: https://pypi.org/project/pydocstyle
     "N",  # see: https://pypi.org/project/pep8-naming
     "S",  # see: https://pypi.org/project/flake8-bandit
+    "C4",  # see: https://pypi.org/project/flake8-comprehensions/
```

Comment:
This one is already in extend-select bellow, is there any difference?

Reply:
No my mistake, removed now :]